Missing JSDoc detail

``` console
yarn lint
```

```

/home/rhys/code/iohk/adrestia-multisig-coordination/test/e2e/jest-setup/docker.ts
  5:8  warning  'path' is defined but never used                         @typescript-eslint/no-unused-vars
  8:7  warning  'CONTAINER_TEMP_DIR' is assigned a value but never used  @typescript-eslint/no-unused-vars

/home/rhys/code/iohk/adrestia-multisig-coordination/src/server/controllers/controllers.ts
  4:1  warning  Missing JSDoc @returns declaration    jsdoc/require-returns
  7:0  warning  Missing JSDoc @param "services" type  jsdoc/require-param-type

/home/rhys/code/iohk/adrestia-multisig-coordination/src/server/db/connection.ts
  5:1  warning  Missing JSDoc @returns declaration            jsdoc/require-returns
  8:0  warning  Missing JSDoc @param "connectionString" type  jsdoc/require-param-type

✖ 6 problems (0 errors, 6 warnings)

Done in 2.08s.
```

Probably best to treat JSDoc lint issues as errors so this is caught it CI

--- a/package.json
+++ b/package.json
@@ -9,7 +9,7 @@
     "test:coverage": "jest --runInBand --coverage",
     "generate-types": "dtsgen --out ./src/types/mcs.d.ts ./src/server/api/openApi.json && prettier --write ./src/types/mcs.d.ts",
     "dev": "ts-node-dev --respawn --transpileOnly -r dotenv/config --watch ./src ./src/server/index.ts",
-    "lint": "eslint \"./test/**/*.ts\" \"./src/**/*.ts\"",
+    "lint": "eslint --max-warnings=0 \"./test/**/*.ts\" \"./src/**/*.ts\"",
     "services:up": "docker-compose up --build",
     "services:down": "docker-compose down",
     "prune-db": "ts-node --transpile-only -r dotenv/config ./src/server/db/prune.ts",

--- a/src/server/controllers/controllers.ts
+++ b/src/server/controllers/controllers.ts
@@ -4,7 +4,8 @@ import messageController, { MessageController } from './message-controller';
 /**
  * Configures all the controllers required by the app
  *
- * @param services App services
+ * @param {Services} services App services
+ * @returns {MessageController} Message controller already configured
  */
 export const configure = (services: Services): MessageController => ({
   ...messageController(services.messageService)

--- a/src/server/db/connection.ts
+++ b/src/server/db/connection.ts
@@ -5,7 +5,8 @@ import { Environment } from '../utils/environment-parser';
 /**
  * Creates a database pool to be used to run queries. No connection will be established.
  *
- * @param connectionString `postgresql://dbuser:secretpassword@database.server.com:3211/mydb`
+ * @param {string} connectionString `postgresql://dbuser:secretpassword@database.server.com:3211/mydb`
+ * @returns {Sequelize} new sequelize connection
  */
 const createPool = (connectionString: string): Sequelize => new Sequelize(connectionString);
 

--- a/test/e2e/jest-setup/docker.ts
+++ b/test/e2e/jest-setup/docker.ts
@@ -2,10 +2,8 @@
 /* eslint-disable no-console */
 import Docker from 'dockerode';
 import { containerExec, imageExists, pullImageAsync } from 'dockerode-utils';
-import path from 'path';
 
 const CONTAINER_IMAGE = 'postgres:11.5-alpine';
-const CONTAINER_TEMP_DIR = '/tmp';
 const CONTAINER_NAME = 'mcs-test';
 
 export const removePostgresContainer = async (): Promise<void> => {
@@ -46,18 +44,10 @@ export const setupPostgresContainer = async (
 
   // Uncomment the following lines if you are going to populate the db with a snapshot
 
-  // await container.putArchive(path.join(__dirname, 'db-snapshot.tar'), {
-  //  path: CONTAINER_TEMP_DIR,
-  //  User: 'root'
-  // });
-  //
   // Wait for the db service to be running (container started event is not enough)
   await containerExec(container, [
     'bash',
     '-c',
     `until psql -U ${user} -d ${database} -c "select 1" > /dev/null 2>&1 ; do sleep 1; done`
   ]);
-  //
-  //  // Execute backup restore
-  //  await containerExec(container, ['bash', '-c', `cat ${CONTAINER_TEMP_DIR}/db.bak | psql -U ${user} ${database}`]);
 };

